website/static/js/comment.js

```diff
@@ -291,6 +511,50 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
+    self.editableContent = ko.computed(function() {
+        var content = self.content();
+        if (!content) {
+            content = '';
+        }
+        var regex = /\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/;
+        var matches = content.match(/\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/g);
+        if (matches) {
+            for (var i = 0; i < matches.length; i++) {
+                var match = regex.exec(matches[i]);
+                var atwho = match[1];
+                var guid = match[3];
+                var mention = match[2];
+
+                content = content.replace(match[0], '<span class="atwho-inserted" contenteditable="false" data-atwho-guid="'+ guid + '" data-atwho-at-query="' + atwho + '">' + atwho + mention + '</span>');
+            }
+        }
+        return content.replace(/\x0D\x0A/g, '<br>');
```

Comment:
Can we use `\r\n` here instead? I think that works, and I think it reads more naturally.

![screen shot 2016-05-02 at 3 59 58 pm](https://cloud.githubusercontent.com/assets/134455/14966158/f809c512-107e-11e6-9b51-f85ce255eac1.png)